### PR TITLE
choco, choco-install, choco-upgrade: add pt_BR translation

### PR DESCRIPTION
**Pull request by brunoeduardodev**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.pt_BR/windows/choco-install.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# choco install
+
+> Instalar um pacote ou mais com Chocolatey.
+> Mais informação: <https://chocolatey.org/docs/commands-install>.
+
+- Instalar um ou mais pacotes separado por espaço:
+
+`choco install {{package(s)}}`
```
Comment (schneiderl) [inline pages.pt_BR/windows/choco-install.md:8]: Nós costumamos traduzir também os parâmetros. Algo do tipo: 
``suggestion
`choco install {{pacote(s)}}`
```

File: pages.pt_BR/windows/choco-install.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Instalar pacotes a partir de um aquivo de configuração personalizado:
+
+`choco install {{path/to/packages.config}}`
+
+- Instalar um aquico específico "nuspec" ou "nupkg":
```
Comment (schneiderl) [inline pages.pt_BR/windows/choco-install.md:14]:
```suggestion
- Instalar um arquivo específico "nuspec" ou "nupkg":
```

File: pages.pt_BR/windows/choco-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Atualizar para uma versão específica de um pacote:
+
+`choco upgrade {{package}} --version {{version}}`
+
+- Atualizar todos pacotes:
+
+`choco upgrade all`
+
+- Atualizr todos, exceto os pacotes separados por vírgula especificados:
```
Comment (schneiderl) [inline pages.pt_BR/windows/choco-upgrade.md:18]: Talvez isso soe melhor?
```suggestion
- Atualizar todos os pacotes, exceto os especificados separados por virgula:
```

File: pages.pt_BR/windows/choco-install.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# choco install
+
+> Instalar um pacote ou mais com Chocolatey.
+> Mais informação: <https://chocolatey.org/docs/commands-install>.
```
Comment (schneiderl) [inline pages.pt_BR/windows/choco-install.md:4]: Por padrão, usamos "Mais informações".
```suggestion
> Mais informações: <https://chocolatey.org/docs/commands-install>.
```

File: pages.pt_BR/windows/choco-upgrade.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# choco upgrade
+
+> Atualizar um ou mais pacotes com Chocolatey.
+> Mais informação: <https://chocolatey.org/docs/commands-upgrade>.
```
Comment (schneiderl) [inline pages.pt_BR/windows/choco-upgrade.md:4]:
```suggestion
> Mais informações: <https://chocolatey.org/docs/commands-upgrade>.
```

File: pages.pt_BR/windows/choco.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# choco
+
+> Uma interface de linha de comando para o gerenciador de pacotes Chocolatey.
+> Veja `choco install`, `choco upgrade` e outras páginas para mais informação.
+> Mais informação: <https://chocolatey.org>.
```
Comment (schneiderl) [inline pages.pt_BR/windows/choco.md:5]:
```suggestion
> Mais informações: <https://chocolatey.org>.
```

**Review by schneiderl — CHANGES_REQUESTED** [review]
Hi,
I've left a few comments for you to take a look at. 
Thanks a lot for your contribution, it is very welcome.

**Comment by brunoeduardodev** [comment]
Thanks, i added your sugestions, sorry for my mistakes, i'm new with open source community and i'm hoping i can help more with this great project

File: pages.pt_BR/windows/choco-install.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`choco install {{path/to/packages.config}}`
+
+- Instalar um arquivo específico "nuspec" ou "nupkg":
+
+`choco install {{path/to/file}}`
+
+- Instalar uma versão específica de um pacote:
+
+`choco install {{package}} --version {{version}}`
```
Comment (navarroaxel) [inline pages.pt_BR/windows/choco-install.md:20]: Remember that you have to translate the arguments to make this easy to read for non-English speakers. e.g. `package` -> `pacote`

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @brunoeduardodev, thanks for your submission. You have some pending changes from @schneiderl. Ask if you have some question.

**Comment by schneiderl** [comment]
@brunoeduardodev Don't be sorry 👍 It is great to see new people getting involved. 

There are still a couple of comments that still unaddressed. Would you mind taking a look at them?

**Comment by brunoeduardodev** [comment]
I"m going to close the pull request and later i'll start a new with all fixes

**Comment by navarroaxel** [comment]
Hey @brunoeduardodev, there is no need to close the PR, we can wait you to resolve the comments or you can convert the PR in a Draft PR you let us know that is a WIP PR.

Also, if you close this PR and open a new one, we lose the track of the comments that we made to you. Do you want I open this PR and we wait for your changes?

**Comment by brunoeduardodev** [comment]
Okay, i think now i covered all the comments, could you take a look?

**Comment by schneiderl** [comment]
@brunoeduardodev thank you for your patience with our review process. It all looks great to me now.

**Review by schneiderl — APPROVED** [review]

**Comment by schneiderl** [comment]
Just wait for @navarroaxel review and I think we can merge this.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**